### DISCUSSION TITLE
How do I reestablish _id links to PCAPs after accidentally deleting "*:so-*" index?

### DISCUSSION THREAD

**Discussion by bfca667** [Security-Onion-Solutions/securityonion#4253] [Unsupported Versions] [answered]
Hi,
I accidentally deleted "*:so-*" default index, then I recreated the index with its old id, but the _id fields no longer appear as links to PCAPs. How do I fix that?

Thanks

**Comment by weslambert** [comment, marked as answer]
Are you referring to the actual index, or the index pattern?  If you deleted the index, then you won't necessarily be able to get that back unless you replayed the data.

**Reply by bfca667** [reply]
Thanks for your reply,

Sorry, I meant the index pattern. I exported the so-* index pattern from a new SO installation and was able to recover the links to pcaps, thehive, etc. Thanks
